iron-menu-behavior.html

```diff
@@ -23,7 +23,8 @@
   Polymer.IronMenuBehaviorImpl = {
 
     properties: {
-
+      _searchText: '',
```

Comment:
If you move it to `created`, this should be ok from the 'changing shapes' perspective. Also, it's the vanilla JS way to handle 'instance variables'. (`created` is roughly called during construction by Polymer.)